Friendly-plans/app/src/androidTest/java/pg/autyzm/friendly_plans/PlanListActivityTest.java

```diff
@@ -151,16 +153,22 @@ public void whenSearchedPlanIsRemovedExpectNoPlansInSearch(){
     public void whenSearchPlanIsRemovedExpectItToBeRemoved(){
         final int testedPlanPosition = 5;
 
-        onView(withId(R.id.et_search_plan_list)).perform(typeText(expectedName + testedPlanPosition));
-        closeSoftKeyboard();
+        //onView(withId(R.id.menuSearch)).perform(click());
+        onView(withId(R.id.menuSearch)).perform(click());
+        onView(isAssignableFrom(EditText.class)).perform(click());
 
+        onView(withId(R.id.menuSearch)).perform(typeText(expectedName + testedPlanPosition));
```

Comment:
I have problem with this test, it is not working for me locally. I think it is related to the size of the device. I think there is a problem with the move of position that is not visible on the sceen.. maybe it would be better to check only the top ones in case the device is too small to show half of the list. (Mine is small) And I think in this tests first lines are not required and could be removed. Worked for me: 

```
        final int testedPlanPosition = 2;
        onView(withId(R.id.menuSearch)).perform(typeText(expectedName + testedPlanPosition));
        onView(withId(R.id.rv_plan_list))
                .perform(RecyclerViewActions
                        .actionOnItemAtPosition(0,
                                clickChildViewWithId(R.id.id_remove_plan)));
        onView(isAssignableFrom(EditText.class)).perform(clearText());

        onView(withRecyclerView(R.id.rv_plan_list)
                .atPosition(testedPlanPosition))
                    .check(matches(hasDescendant(withText(expectedName
                        + (testedPlanPosition + 1)))));
```